benchmarks/src/main/terraform/aws/modules/emr/main.tf

```diff
@@ -0,0 +1,162 @@
+##
+## Provider Definition
+##
+provider "aws" {
+  region = "${var.aws_region}"
+  access_key = "${var.aws_access_key}"
+  secret_key = "${var.aws_secret_key}"
+}
+
+data "aws_vpc" "main" {
+  id = "${var.aws_vpc_id}"
+}
+
+data "aws_subnet" "main" {
+  id = "${var.aws_subnet_id}"
+}
+
+resource "aws_key_pair" "key" {
+  public_key = "ssh-rsa ${var.aws_ssh_public_key == "" ? "AAAAB3NzaC1yc2EAAAADAQABAAABAQC0eX0fhy3WTIHF13DuSTHBFjLzKRssFRrW6e2B+/9Oh2Ua/zsEoIeLyX5YtPAqeR22DVJBA+sOvKMQnenAVUa0XG7y6rzEPgugqWNv6NVsFgbgHMfWpRYcuPuOo42T0AQD/9rLViyAzy6lRDid3gpN3PkSBhDLGPEZYs9Lzucawm2FZV92/9u5CxgvRZBAAIrWtgHwGpos3mVuisNxHjH3uEv0B43NzN5hJfBYiEyHhwi2eyjTuDFvVQ8rywcrDZ+aR2BTRX+roR7eVq7isjyOq41qy+pRsRLl8/9ULA6HvDYyozN+jCd5xhFJHTMG1IInapIUcRewtqzsgA9XggyT" : var.aws_ssh_public_key}"
+}
+
+resource "aws_s3_bucket" "deployment_bucket" {
+  acl = "public-read"
+  force_destroy = true
+  tags = {
+    Name= "SparklingWaterBenchmarksDeploymentBucket"
+  }
+}
+
+resource "aws_s3_bucket_policy" "read_objects" {
+  bucket = "${aws_s3_bucket.deployment_bucket.id}"
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Id": "ReadObjectsPolicy",
+  "Statement": [
+    {
+      "Sid": "PublicReadForGetBucketObjects",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::${aws_s3_bucket.deployment_bucket.bucket}/public-read/*"
+    }
+  ]
+}
+POLICY
+}
+
+resource "aws_s3_bucket_object" "benchmarks_jar" {
+  bucket = "${aws_s3_bucket.deployment_bucket.bucket}"
+  key = "benchmarks.jar"
+  acl = "private"
+  source = "${var.sw_package_file}"
+}
+
+resource "aws_s3_bucket_object" "run_benchmarks_script" {
+  bucket = "${aws_s3_bucket.deployment_bucket.id}"
+  key    = "run_benchmarks.sh"
+  acl = "private"
+  content = <<EOF
+
+  #!/bin/bash
+  set -x -e
+
+  function runBenchmarks {
+    spark-submit \
+      --class ai.h2o.sparkling.benchmarks.Runner \
```

Comment:
I'm sure you have tested it, but I can't see the sparkling water `jar` being passed to Spark here

Reply:
[This fat jar](https://github.com/h2oai/sparkling-water/pull/1428/files#diff-dffa08e9205b1dcdae26fb7ad9c34f37R80) should contain everything

Reply:
Yup, I just don's see the jar to be passed to spark-submit